src/protocol/tx-validity.md

```diff
@@ -33,9 +33,9 @@ Read-only access list:
 Write-destroy access list:
 
 - For each [input `InputType.Coin`](../tx-format/input.md#inputcoin)
-  - The [UTXO ID](../identifiers/utxo-id.md) `(txID, outputIndex)`
+  - The [UTXO ID](../identifiers/utxo-id.md) `(tx_id, output_index)`
 - For each [input `InputType.Contract`](../tx-format/input.md#inputcontract)
-  - The [UTXO ID](../identifiers/utxo-id.md) `(txID, outputIndex)`
+  - The [UTXO ID](../identifiers/utxo-id.md) `(tx_id, output_index)`
```

Comment:
Our specification uses camelCase everywhere. It is better to keep it in one style=)

Reply:
I can switch it back to camel case.

I would warn that, since we are using python (or some language that is obviously python-esque) as our pseudocode, we may create some confusion to readers who expect snake case or depend on other established heuristics in order to comprehend code.

If we want camel case, I would suggest distancing ourselves from python and using something else that allows us to be consistent with the language rules.

Reply:
Yes, we can update this in a separate PR if we decide this is worth addressing. For now, I've reverted it back to camelCase for the use of member variables (some functions and stack variables local to pseudo functions are kept snake case)